org.mwc.debrief.pepys/src/main/java/org/mwc/debrief/pepys/model/db/config/ConfigurationReader.java

```diff
@@ -0,0 +1,120 @@
+package org.mwc.debrief.pepys.model.db.config;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Scanner;
+
+import junit.framework.TestCase;
+
+public class ConfigurationReader {
+
+	public static class ConfigurationReaderTest extends TestCase {
+
+		public void testConfigurationRead() {
+			final String testConfiguration = "[database]\n" + "db_username = abcdef\n" + "db_password = _abcdef_\n"
+					+ "db_host = localhost\n" + "db_port = 5432\n" + "db_name = pepys2\n" + "[archive]\n"
+					+ "user = abcdef\n" + "password = abcdef";
+			final InputStream stream = new ByteArrayInputStream(testConfiguration.getBytes());
+			final DatabaseConfiguration databaseConfigurationTest = new DatabaseConfiguration();
+			parseConfigurationFile(databaseConfigurationTest, stream);
+
+			// Check Database Categories
+			final HashMap<String, String> databaseCategory = databaseConfigurationTest.getCategory("database");
+			final HashMap<String, String> archiveCategory = databaseConfigurationTest.getCategory("archive");
+
+			assertTrue("databaseCategory - size", databaseCategory.size() == 5);
+			assertTrue("archiveCategory - size", archiveCategory.size() == 2);
+
```

Comment:
Could we have difference values for `db_username` and `db_password`?  

The code is _slightly_ weaker if it's testing for the same string twice.  We _may_ get username instead of password, but still get the same results string.

So, `abcdef` for username is fine, but password should be different, like `uvwxyz`.